tests/unit/OptionsListUtilsTest.js

```diff
@@ -1056,6 +1056,373 @@ describe('OptionsListUtils', () => {
         expect(largeWrongSearchResult.categoryOptions).toStrictEqual(largeWrongSearchResultList);
     });
 
+    it('getNewChatOptions() for tags', () => {
+        const search = 'ing';
+        const emptySearch = '';
+        const wrongSearch = 'bla bla';
+        const recentlyUsedTags = ['Engineering', 'HR'];
+
+        const selectedOptions = [
+            {
+                name: 'Medical',
+            },
+        ];
+        const smallTagsList = {
+            Engineering: {
+                enabled: false,
+                name: 'Engineering',
+            },
+            Medical: {
+                enabled: true,
+                name: 'Medical',
+            },
+            Accounting: {
+                enabled: true,
+                name: 'Accounting',
+            },
+            HR: {
+                enabled: true,
+                name: 'HR',
+            },
+        };
+        const smallResultList = [
+            {
+                title: '',
+                shouldShow: false,
+                indexOffset: 0,
+                data: [
+                    {
+                        text: 'Engineering',
+                        keyForList: 'Engineering',
+                        searchText: 'Engineering',
+                        tooltipText: 'Engineering',
+                        isDisabled: true,
```

Comment:
Can you confirm that we need to show disabled tags if they they are not parents? Because, for categories, we should hide disabled ones.

Reply:
What do you mean parents? If you mean what you have in categories like 'Food > Meat', there isn't that in tags

Reply:
Good. It is much easier 😅
But, anyway, should the app show disabled tags or hide?